lms/views/basic_lti_launch.py

```diff
@@ -285,7 +285,7 @@ def unconfigured_basic_lti_launch(self):
         ).authorization_param(self.request.lti_user)
 
         self.context.js_config.enable_content_item_selection_mode(
-            form_action=self.request.route_url("configure_assignment"),
+            form_action=self.request.route_url("lti_launches"),
```

Comment:
Because the creation API has already created the assignment no need to configure it. Not sure if it has any other implications.

Reply:
I think this is where more frontend work would be needed here. On this branch when I select a file the frontend both calls the create assignment API and then also makes a form submission to the `/lti_launches` endpoint. This form submission is unnecessary. The frontend should just call the create assignment API, that's all that's necessary to configure the assignment. But _then_ the frontend needs to go ahead and launch the assignment (show the document and the client etc), but without making any form submissions or doing any page reloads. The frontend should have all the information it needs to launch the assignment (e.g. the document URL) from the create assignment API's response